common/culture/cultures/ancient.txt

```diff
@@ -0,0 +1,250 @@
+﻿ancient_group = { 
```

Comment:
Merging the master was a bad idea. You should merge the adaptation branch into here.

Reply:
Fixed it.